types & grammar/ch2.md

```diff
@@ -49,7 +49,7 @@ a.length;	// 3
 
 Такой код может привести к странному поведению "пустых ячеек" оставленных между элементами массива. Пустой слот со значением `undefined` внутри, ведет себя не так же как явно объявленный элемент массива (`a[1] = undefined`). Подробнее в главе 3 "Массивы".
 
-Массивы `array`s проиндексированы числами (как и ожидается), но хитрость в том, что они могут иметь индекс в виде  строки `string` ключ/свойство могут быть добавлены к массиву (но такие свойства не будут посчитаны в длине массива `length`):
+Массивы `array`s проиндексированы числами (как и ожидается), но хитрость в том, что они также являются объектами, которые могут иметь `string` ключи/свойства, добавленные к ним (но такие свойства не будут посчитаны в длине массива `length`):
```

Comment:
[Original](https://github.com/getify/You-Dont-Know-JS/blame/master/types%20%26%20grammar/ch2.md#L52):

> but the tricky thing is that they also are objects that can have `string` keys/properties added to them